internal/util/crypto.go

```diff
@@ -243,7 +249,10 @@ func EncryptVolume(ctx context.Context, devicePath, passphrase string) error {
 // OpenEncryptedVolume opens volume so that it can be used by the client.
 func OpenEncryptedVolume(ctx context.Context, devicePath, mapperFile, passphrase string) error {
 	DebugLog(ctx, "Opening device %s with LUKS on %s", devicePath, mapperFile)
-	_, _, err := LuksOpen(devicePath, mapperFile, passphrase)
+	_, stderr, err := LuksOpen(devicePath, mapperFile, passphrase)
+	if err != nil {
+		WarningLog(ctx, "failed to open LUKS device %q: %s", devicePath, stderr)
```

Comment:
this should be an error log not a warning log?

Reply:
I'd leave that up to the caller. The caller can log the returned error with a level it deems appropriate. Only because it is a failure, I didn't think Info-level would be suitable.